monai/data/box_utils.py

```diff
@@ -0,0 +1,360 @@
+# Copyright (c) MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from copy import deepcopy
+from typing import Sequence, Union
+
+import numpy as np
+import torch
+
+from monai.config.type_definitions import NdarrayOrTensor
+
+# from monai.utils.misc import ensure_tuple_rep
+from monai.utils.module import look_up_option
+from monai.utils.type_conversion import convert_to_numpy, convert_to_tensor
+
+# We support several box modes, i.e., representation of a bounding box
+CORNER_CORNER_MODE = ["xyxy", "xyzxyz"]  # [xmin, ymin, xmax, ymax] and [xmin, ymin, zmin, xmax, ymax, zmax]
+XXYYZZ_MODE = ["xxyy", "xxyyzz"]  # [xmin, xmax, ymin, ymax] and [xmin, xmax, ymin, ymax, zmin, zmax]
+CORNER_SIZE_MODE = ["xywh", "xyzwhd"]  # [xmin, ymin, xsize, ysize] and [xmin, ymin, zmin, xsize, ysize, zsize]
+CENTER_SIZE_MODE = [
+    "ccwh",
+    "cccwhd",
+]  # [xcenter, ycenter, xsize, ysize] and [xcenter, ycenter, zcenter, xsize, ysize, zsize]
+
+STANDARD_MODE = CORNER_CORNER_MODE  # standard box modes supported by all the box util functions
+SUPPORT_MODE = (
+    CORNER_CORNER_MODE + XXYYZZ_MODE + CORNER_SIZE_MODE + CENTER_SIZE_MODE
+)  # supported box modes for some box util functions
+
+# TO_REMOVE = 0 if in 'xxyy','xxyyzz' mode, the bottom-right corner is not included in the box,
+#      i.e., when xmin=1, xmax=2, we have w = 1
+# TO_REMOVE = 1  if in 'xxyy','xxyyzz' mode, the bottom-right corner is included in the box,
+#       i.e., when xmin=1, xmax=2, we have w = 2
+# Currently only TO_REMOVE = 0 has been tested. Please use TO_REMOVE = 0
+TO_REMOVE = 0  # xmax-xmin = w -TO_REMOVE.
```

Comment:
As mentioned in the comment, it is set to be fixed value for now. So we would not want the users to change it currently. In the future, we would test TO_REMOVE = 1 and let the user to change it.

Reply:
OK, I see.
Could you please help slightly adjust the comments? because usually, we don't suppose users to use MONAI source code in their project, instead, they just install MONAI and call your APIs. Comments can be:
```
Currently, only `TO_REMOVE = 0` is supported
```
What do you think?

Thanks.